src/components/__tests__/IspSelectWithIncidents-test.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import chai, { expect } from 'chai';
+
+import chaiEnzyme from 'chai-enzyme';
+import { IspSelectWithIncidents } from 'components';
+
+chai.use(chaiEnzyme());
+
+describe('components', () => {
+  describe('IspSelectWithIncidents', () => {
+    it('selecting multiple ISPs', () => {
+
+      let selected = [];
+
+      const mockChange = function(ispId) { selected.push(ispId); };
+
+      // Sample data for this test
+      const isps = {
+        AS10774x: { client_asn_name: 'AT&T', client_asn_number: 'AS10774x' },
+        AS10796x: { client_asn_name: 'Time Warner Cable', client_asn_number: 'AS10796x' },
+        AS11486x: { client_asn_name: 'Verizon', client_asn_number: 'AS11486x' },
+        AS10507: { client_asn_name: 'Sprint Personal Communications Systems', client_asn_number: 'AS10507' },
+      };
+        
+      const incidentData = {};
+
+      const onChangeIncidentASN = function(){return;};
```

Comment:
Make sure you have spaces between your brackets throughout your code (https://eslint.org/docs/rules/space-in-brackets)